site/docs/voice/bxml/transfer.mdx

```diff
@@ -25,6 +25,7 @@ There is no text content available to be set for the `<Transfer>` verb.
 | Attribute | Description |
 |:----------|:------------|
 | transferCallerId | (optional) The caller ID to use when the call is transferred, if different. Must be in E.164 format (e.g. +15555555555) or be one of the following strings `Restricted`, `Anonymous`, `Private`, or `Unavailable`.<br/><br/> Note: Leave blank to pass along the number of the remote party. |
+| transferCallerDisplayName | (optional) The caller display name to use when the call is transferred, a single line of text, with no embedded newline characters, may not exceed 256 characters. The characters allowed in the transferCallerDisplayName are specified in [`RFC 3261 - Section 25.1`](https://www.rfc-editor.org/rfc/rfc3261.html#page-228).|
```

Comment:
I feel like pointing to an RFC is too technical, I would simplify a bit like this.
```suggestion
| transferCallerDisplayName | (optional) The caller display name to use when the call is transferred. May not exceed 256 characters or contain control characters such as new lines. |
```